modules/sdk-coin-osmo/test/unit/transactionBuilder/StakingWithdrawRewardsBuilder.ts

```diff
@@ -0,0 +1,107 @@
+import { toHex, TransactionType } from '@bitgo/sdk-core';
+import { fromBase64 } from '@cosmjs/encoding';
+import should from 'should';
+
+import * as testData from '../../resources/osmo';
+import { getBuilderFactory } from '../getBuilderFactory';
+
+describe('Osmo WithdrawRewards txn Builder', () => {
```

Comment:
remove this file, we are not supporting WithdrawReward transaction from UI.

Reply:
better keep it, at least we will have support in SDK that people might use directly